webdriver-spec.html

```diff
@@ -969,7 +969,7 @@
  <tr>
   <td>POST</td>
   <td>/session/{session id}/alert/text</td>
-  <td><a>Send Alert Text</a></td>
```

Comment:
Because it no longer implies that ti should use the send keys algorithm that is used for input form controls for entering text into prompts.

This specification never outlined using that algorithm for prompts, but Marionette for example uses it for typing into prompts.  This is completely unnecessary and can’t be supported in other UAs.